python/utils.py

```diff
@@ -244,4 +244,44 @@ def check_digit_gen(valid=True, ULI=None):
 	else:
 		return str(check_digit+6).zfill(2)[:2] #return a bad check digit (used in edit testing)
 
+def validate_state_codes(path, data_file):
+	"""Parses through an existing test file and replaces the state code 
```

Comment:
can you specify what sort of data file this is? Maybe name it lar_file?

Reply:
Change has been made in a new commit.